monix-eval/shared/src/main/scala/monix/eval/Task.scala

```diff
@@ -2404,7 +2404,7 @@ object Task extends TaskInstancesLevel1 {
     * Switch to [[Task.evalAsync]] if you wish the old behavior, or combine
     * [[Task.eval]] with [[Task.executeAsync]].
     */
-  def apply[A](@deprecatedName('f) a: => A): Task[A] =
```

Comment:
Why was that annotation removed?

Reply:
The `Symbol` version is deprecated in 2.13 in the `String` version does not exist in 2.11